docs/advanced-recipes/custom-useatom-hooks.mdx

```diff
@@ -0,0 +1,51 @@
+# Costuom useatom Hooks
+
+This page shows the ways of creating different utility functions. Utility functions save your time on coding, and you can preserve your base atom for other usage.
+
+## utils
+
+### useSelectAtom
+
+```ts
+import { useAtom } from 'jotai'
+import { selectAtom } from 'jotai/utils'
+
+export function useSelectAtom(anAtom, keyFn) {
+    return useAtom(selectAtom(anAtom, keyFn));

```

Comment:
Please add a note that in this case `keyFn` must be stable, either define outside render or warp with `useCallback`.